source/documentation/common/backup-restore/proc-Restoring_the_Backup_on_a_New_Self-hosted_Engine.adoc

```diff
@@ -93,7 +93,16 @@ The script creates the virtual machine. This can take some time if the {engine-a
 +
 [NOTE]
 ====
-If the host becomes non operational and the deployment fails (due to missing required network or similar problem), you can repeat the process, and answer _YES_ to the option _Pause the execution after adding this host to the engine?_. Pausing the process allows you to connect to the restored engine and manually review and remediate the configuration before proceeding with the deployment.
+If the host becomes non operational and the deployment fails due to a missing required network or a similar problem, deployment will pause and a message such as the following will be displayed:
```

Comment:
I deliberately did not keep 'the deployment fails', because it does not - and that's a significant part of bug 1893385 - see especially comments 19-20 there. From the users' POV, if you enter this flow, there shouldn't be any '[ERROR]' - you get a prompt saying the host is non-operational and you should manually fix, all green and nice.